consumer/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3.7"
+services:
+  web:
+    network_mode: host
+    build:
+      context: .
+    environment:
+      DATABASE_URL: "postgres://dataservices:insecure@${HOSTNAME}:5416/dataservices"
+      SCHEMA_URL: https://356c76835e424b968ed6d654c51204f0.objectstore.eu/schemas-development/datasets/
+      BOOTSTRAP_SERVERS: ${HOSTNAME}:29092
+      TOPIC: events
+      GROUP_ID: python_jan_example_group_1
```

Comment:
die GROUP_ID lijkt haast een eigen test zo.. 

Wellicht deze file ook in `scripts/...` of een `tools/events` map zetten? Doet een beetje gek aan zo op hoofd niveau.